coderd/presets.go

```diff
@@ -38,12 +39,21 @@ func (api *API) templateVersionPresets(rw http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	getPrebuildInstances := func(desiredInstances sql.NullInt32) *int {
```

Comment:
this name implies fetching from somewhere, but it only does a type conversion. perhaps consider a more descriptive name?